cmd/oras/internal/option/packer.go

```diff
@@ -114,12 +114,17 @@ func (opts *Packer) parseAnnotations(cmd *cobra.Command) error {
 	return nil
 }
 
-// decodeJSON decodes a json file v to filename.
+// decodeJSON decodes file contents into json.
 func decodeJSON(filename string, v interface{}) error {
```

Comment:
We need to make `error` a named return value
```suggestion
func decodeJSON(filename string, v interface{}) (err error) {
```

Reply:
Done.